src/shared_modules/rsync/src/rsyncImplementation.cpp

```diff
@@ -175,7 +185,20 @@ void RSyncImplementation::registerSyncId(const RSYNC_HANDLE handle,
 
         }
     };
-    ctx->m_msgDispatcher.addCallback(messageHeaderID, registerCallback);
+
+    auto syncInterval { std::chrono::seconds(0) };
+    const auto itInterval { syncConfiguration.find("minimal_sync_interval") };
+
+    if (itInterval != syncConfiguration.end())
+    {
+        const auto intervalValue { itInterval->get<int32_t>() };
+        syncInterval = std::chrono::seconds(intervalValue > 0 ? intervalValue : 0);
```

Comment:
```suggestion
        syncInterval = intervalValue > 0 ? std::chrono::seconds(intervalValue);
```
This will unify the default value set on L189